docs/helm/kratos.md

```diff
@@ -27,6 +31,59 @@ kratos:
       default:
         - dolore occaecat nostrud Ut
         - sit et commodoaute ut voluptate consectetur Duis
+    identity:
+      default_schema_id: default
+      schemas:
+        - id: default
+          url: file:///etc/config/identity.default.schema.json
+    courier:
+      smtp:
+        connection_uri: smtps://test:test@mailslurper:1025/?skip_ssl_verify=true
+    selfservice:
+      default_browser_return_url: http://127.0.0.1:4455/
+  automigration:
```

Comment:
Automigration is set to job by default, if we want to specify here which values have to be enabled, the we can shorten that to:

```yaml
automigration:
  enabled: true
```

However, this doesn't have to be enabled, as you can and sometimes might want to run the migrations manually

Reply:
I think it's worth setting it to true to provide an example that would work without modification. Or else user would likely see database error in pod. 
I would remove the other configs under automigration.